src/core/public/chrome/ui/header/header_breadcrumbs.scss

```diff
@@ -0,0 +1,35 @@
+/*
+* Copyright OpenSearch Contributors
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+.headerBreadcrumbs {
```

Comment:
should css prefixes for the new header be the same?

Reply:
We are moving the component to OUI. Will take care of it there.